huaweicloud/services/as/resource_huaweicloud_as_bandwidth_policy.go

```diff
@@ -435,3 +449,95 @@ func buildUpdateBandwidthPolicyScheduledPolicyChildBody(d *schema.ResourceData)
 
 	return params
 }
+
+func GetBandwidthPolicy(client *golangsdk.ServiceClient, policyId string) (interface{}, error) {
+	httpUrl := "autoscaling-api/v2/{project_id}/scaling_policy/{scaling_policy_id}"
+	getPath := client.Endpoint + httpUrl
+	getPath = strings.ReplaceAll(getPath, "{project_id}", client.ProjectID)
+	getPath = strings.ReplaceAll(getPath, "{scaling_policy_id}", policyId)
+	getOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	resp, err := client.Request("GET", getPath, &getOpt)
+	if err != nil {
+		return nil, err
+	}
+
+	respBody, err := utils.FlattenResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return respBody, nil
+}
+
+func updateBandwidthPolicyStatus(ctx context.Context, client *golangsdk.ServiceClient, t time.Duration,
+	policyId, action string) error {
+	updateStatusHttpUrl := "autoscaling-api/v1/{project_id}/scaling_policy/{scaling_policy_id}/action"
+	updateStatusPath := client.Endpoint + updateStatusHttpUrl
+	updateStatusPath = strings.ReplaceAll(updateStatusPath, "{project_id}", client.ProjectID)
+	updateStatusPath = strings.ReplaceAll(updateStatusPath, "{scaling_policy_id}", policyId)
+
+	updatePolicyStatusOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		JSONBody: utils.RemoveNil(map[string]interface{}{
+			"action": utils.ValueIgnoreEmpty(action),
```

Comment:
What is the purpose of adding `utils.ValueIgnoreEmpty` and removeNil in this place?